src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Schemas/ControlToken.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.LanguageServerProtocol.Protocol;
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Schemas
+{
+    internal class ControlToken
+    {
+        // Control name
+        public string Name { get; set; }
+
+        // Representing the range of the control token
+        public LinkedList<List<uint>> EncodedTokenIndices { get; set; }
```

Comment:
Why not use Range class here which is language server compliant? Also linked list here looks a bit weird choice. You certainly don't need to lots of elements towards the end of the list.

Reply:
I agree, I can change the Linkedlist since it may not be necessary. But Range class expects position in int. But I believe for our use case restricting to uint is a good idea.